diff --git a/dist/index.d.ts b/dist/index.d.ts
index 0ddb2bc..023aa68 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -297,7 +297,7 @@ declare function defineSimpleViewModel<const T extends StandardJSONSchemaV1, con
 //#region src/action_view_model.d.ts
 type AnyAction = (fnArg: any) => void;
 declare class ActionModel<Fn extends AnyAction> {
-  action!: Fn;
+  action: Fn;
 }
 /**
 * Defines a ViewModel whose only attribute is a direct `~action` that assigned to `ActionModel`'s `action`.
diff --git a/dist/index.js b/dist/index.js
index ea05249..06025e3 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -291,7 +291,7 @@ function defineSimpleViewModel(schema, options) {
 //#endregion
 //#region src/action_view_model.ts
 var ActionModel = class {
-	action;
+	action = void 0;
 };
 /**
 * Defines a ViewModel whose only attribute is a direct `~action` that assigned to `ActionModel`'s `action`.
diff --git a/src/action_view_model.ts b/src/action_view_model.ts
index 4c91f54..5f8ff38 100644
--- a/src/action_view_model.ts
+++ b/src/action_view_model.ts
@@ -10,7 +10,7 @@ import type { AR } from "./attribute_return.ts";
 type AnyAction = (fnArg: any) => void;
 
 export class ActionModel<Fn extends AnyAction> {
-  action!: Fn;
+  action: Fn = undefined!;
 }
 
 /**
